core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -85,7 +85,19 @@ private[spark] class CoarseGrainedExecutorBackend(
 
     logInfo("Connecting to driver: " + driverUrl)
     try {
-      if (PlatformDependent.directBufferPreferred() &&
+      // The following logic originally comes from the constructor of
```

Comment:
Shall add this logic into a NettyUitls function?

Reply:
I'm afraid not, `NettyUtils` located at `network-common`, it can not access `SparkConf` which in `core`

Reply:
How about NettyUtils just accept boolean inputs, not config

Reply:
Updated, @Ngone51 would you please take a look again?